bigquery/dump/load-data-by-POST.py

```diff
@@ -0,0 +1,104 @@
+import sys
```

Comment:
Isn't the 'dump' dir used for just copying something outside to the repo?
Do we need this file?